templates/matrix.hbs

```diff
@@ -17,7 +17,7 @@
         </thead>
       <tbody>
     {{#each matrix.rows}}
-      <tr class={{atRisk cells}}>
```

Comment:
I'm fine if we keep these here. I just didn't want that to be the only way to find them.

Reply:
because At Risk is just another matrix it results in all of the rows in At Risk having opacity 0.6 so I think it's better just to do the At Risk section or change the way at risk rows are highlighted.

Reply:
Addressed this with a thin outline: https://github.com/digitalbazaar/mocha-w3c-interop-reporter/pull/49/commits/130d0d5fcf3cdcb0c667b6b7f25b2d1425108cbc